contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -39,11 +39,14 @@ contract TrueCreditAgency is UpgradeableClaimable {
 
     mapping(ITrueFiPool2 => mapping(address => uint8)) public creditScore;
     mapping(ITrueFiPool2 => mapping(address => uint256)) public borrowed;
+    mapping(ITrueFiPool2 => mapping(address => uint256)) public nextRepaymentTimestamp;
```

Comment:
I'm sorry to make you rename this yet again, but we should clarify that this is for interest repayments. Let's call this `nextInterestRepaymentTimestamp`, or shorten it if you'd like (`nextInterestRepayTime`?). Update elsewhere as well.

We'll also need to track biannual full repayments, but that can be done in another PR after this one.